upgrade-notes/backend_20240717_111911.md

```diff
@@ -0,0 +1,79 @@
+#### limited item counts in sliders ([#3244](https://github.com/shopsys/shopsys/pull/3244))
+
+-   `Shopsys\FrameworkBundle\Model\Product\Accessory\ProductAccessoryFacade` class was changed:
+    -   `getTopOfferedAccessories()` method was renamed to `getOfferedAccessories()`
+-   `Shopsys\FrameworkBundle\Model\Product\Accessory\ProductAccessoryRepository` class was changed:
+    -   `getTopOfferedAccessories()` method was renamed to `getOfferedAccessories()`
+-   `Shopsys\FrameworkBundle\Model\Product\BestsellingProduct\BestsellingProductFacade` class was changed:
+    -   `getAllOfferedBestsellingProducts()` method was renamed to `getOfferedBestsellingProducts()` and changed its interface
+        ```diff
+            public function getOfferedBestsellingProducts(
+                // ...
+        +       int $limit
+        ```
+-   `Shopsys\FrameworkBundle\Model\Product\BestsellingProduct\CachedBestsellingProductFacade` class was changed:
+    -   `getAllOfferedBestsellingProductIds()` method was renamed to `getOfferedBestsellingProductIds()` and changed its interface
+        ```diff
+            public function getOfferedBestsellingProductIds(
+                // ...
+        +       int $limit
+        ```
+-   `Shopsys\FrameworkBundle\Model\Product\Elasticsearch\ProductExportRepository` class was changed:
+    -   `__construct()` method changed its interface:
+        ```diff
+            public function __construct(
+                // ...
+        +        protected readonly ProductFrontendLimitProvider $productFrontendLimitProvider,
+        ```
+    -   `extractAccessoriesIds()` method changed its interface
+        ```diff
+            public function getOfferedBestsellingProductIds(
+                // ...
+        +       int $domainId
+        +       ?int $limit
+        ```
+-   `Shopsys\FrameworkBundle\Model\Product\TopProduct\TopProductFacade` class was changed:
+    -   `getAllOfferedProducts()` method was renamed to `getOfferedProducts()` and changed its interface
+        ```diff
+            public function getOfferedProducts(
+                // ...
+        +       ?int $limit
```

Comment:
the type is not in the class. please either add a type or remove a type from the upgrade note